csharp/ql/src/semmle/code/csharp/dataflow/LibraryTypeDataFlow.qll

```diff
@@ -1876,11 +1884,11 @@ class SystemRuntimeCompilerServicesConfiguredTaskAwaitableTFlow extends LibraryT
     // var result = awaiter.GetResult();
     c = this.getGetAwaiterMethod() and
     source = TCallableFlowSourceQualifier() and
-    sourceAp = AccessPath::empty() and
-    sink = TCallableFlowSinkReturn() and
-    sinkAp =
+    sourceAp =
```

Comment:
hmm, I don't know how I messed this up with the explicitly added comment above it. Thanks.